boards/modalai/voxl2/target/voxl-px4-set-default-parameters.config

```diff
@@ -12,9 +12,8 @@ param set IMU_GYRO_RATEMAX 800
 param set EKF2_IMU_POS_X 0.027
 param set EKF2_IMU_POS_Y 0.009
 param set EKF2_IMU_POS_Z -0.019
-param set EKF2_EV_CTRL 15
 param set EKF2_EV_DELAY 5
-param set EKF2_GPS_CTRL 0
+param set EKF2_AID_MASK 280
```

Comment:
Okay, yes, thanks! That file needs to be refreshed. But we have a separate process to come in and set all the parameters as needed for each of our drones so that file really isn't relied upon for anything critical.

Reply:
This repo has our actual parameter settings if you are interested: https://gitlab.com/voxl-public/voxl-sdk/utilities/voxl-px4-params/-/tree/master/params/v1.14?ref_type=heads